### PR TITLE
Nasi goreng

### PR DESCRIPTION
**Pull request by NicMoe**
Adding nasi goreng recipe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ We currently have 27 recipes available. For recipes in development, check out [t
 
 ### Entrees
 
+- [Nasi goreng](entrees/nasi-goreng.md)
```
Comment (diego-codes) [inline README.md:38]: Can you run the python script to update this document instead please?

File: entrees/nasi-goreng [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Nasi goreng
+
+This recipe is based off [this Food Network recipe](https://www.foodnetwork.com/applink/recipes/food-network-kitchen/nasi-goreng-5482485).
```
Comment (diego-codes) [inline entrees/nasi-goreng]: It might be good to add a description of what Nasi goreng is.

File: entrees/nasi-goreng [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Ingredients
+
+- 1.5 cups (dry) or 3 cups (cooked and cooled) long grain rice
+- 2 tablespoons soy sauce
+- 1 tablespoon fish sauce
+- 2 teaspoons sambal oelek, plus more for serving
+- 3 tablespoons peanut oil
+- 3 large eggs, beaten
+- 12 ounces small shrimp, peeled and deveined
```
Comment (diego-codes) [inline entrees/nasi-goreng]: Should the shrimp be raw or precooked? I assume raw

**Review by diego-codes — APPROVED** [review]
I have some suggestions, otherwise it looks good!

File: entrees/nasi-goreng (hunk 2 above)
Comment (NicMoe) [reply entrees/nasi-goreng]: Added

File: README.md (hunk 1 above)
Comment (NicMoe) [reply README.md:38]: Done

**Pull request merged**